gsuite_reports_rules/gsuite_drive_overly_visible.yml

```diff
@@ -7,7 +7,7 @@ LogTypes:
   - GSuite.Reports
 Tags:
   - GSuite
-Severity: Medium
```

Comment:
I think this one should be `Info` and the other one should be `Low`. This one is just saying someone interacted with a public document, the other rule is looking for newly public documents.